services/openshift-service-impl/src/test/java/org/kontinuity/catapult/service/openshift/impl/OpenShiftServiceTestBase.java

```diff
@@ -27,8 +33,25 @@
 
     private final Collection<OpenShiftProject> createdProjects = new ArrayList<>();
 
+    private final URI pipelineTemplate;
+    //private final InputStream pipelineTemplate = Thread.currentThread().getContextClassLoader().getResourceAsStream("test-template.yml");
+    
     protected abstract OpenShiftService getOpenShiftService();
     
+	public OpenShiftServiceTestBase() {
```

Comment:
Why do init in the ctor?  Subclasses may not opt to explicitly call `super()` and this may miss getting invoked.  How about in `@Before`; does that work equally well?